microsoft-edge/devtools-guide-chromium/javascript/ado-symbol-server.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Debug original code using the Azure Artifacts symbol server
```

Comment:
I think the title should be something more along the lines of "Retrieve source maps using Azure Artifacts symbol server" or something like that. In Edge 100, this will work with the performance tool as well -- not leading us to "Debug original code"

I don't know. Don't love this title but it's not the end of the world either.

Reply:
Good perspective, thanks. I'm using this title now.

Reply:
The TOC title is allowed to remove words vs. the top-of-page title.
The TOC title is not allowed to use different words than the top-of-page title.